src/otx/algo/classification/deit_tiny.py

```diff
@@ -78,6 +78,9 @@ class DeitTinyForHLabelCls(ExplainableDeit, MMPretrainHlabelClsModel):
     """DeitTiny Model for hierarchical label classification task."""
 
     def __init__(self, num_classes: int, num_multiclass_heads: int, num_multilabel_classes: int) -> None:
+        self.num_multiclass_heads = num_multiclass_heads
```

Comment:
This information is added to use the `Model.num_multiclass_heads` and `Model.num_multilabel_classes` information at the `Metric`.